### DISCUSSION TITLE
How to edit php configuration?

### DISCUSSION THREAD

**Discussion by bit-config** [nextcloud/all-in-one#3304] [Questions] [answered]
I Googled a little and realized that I needed to edit some limits
https://github.com/nextcloud/all-in-one/discussions/2523
like here. But I can't figure out how to do it in a container

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1335

**Reply by bit-config** [reply]
There are no performance problems, on a virtual machine with 16GB and SMB disks, everything works fine
Only a couple of times there were php related errors that are solved as I understand in php editing

**Reply by szaimen** [reply]
Adding more ram will automatically scale max php workers so that is why I linked the above article.

**Reply by bit-config** [reply]
This is not a solution for me. There are only obvious things in that information.
I increased the memory to 16GB, but it still changes values.
Error [06-Sep-2023 10:45:49] WARNING: [pool www] server reached max_children setting (158), consider revising it as it was and remains.

**Reply by szaimen** [reply]
Can you disable collabora and check if that fixes it?

**Reply by bit-config** [reply]
I'll try next time, now I've rebooted for a solution.
Also a mistake
Doctrine\DBAL\Exception: Failed to connect to the database: An exception occurred in the driver: SQLSTATE[08006] [7] connecting to the server at "nextcloud-aio-database" (172.18.0.4), port 5432 failure: FATAL: the remaining connection slots are reserved for superuser connections without replication in /var/www/html/lib/private/DB/Connection.php:140

**Reply by bit-config** [reply]
Basically all the errors appeared after the last update

**Comment by szaimen** [comment, marked as answer]
> How to edit php configuration?

You cannot, at least not in a permanent way.
